tests/e2e/client/Makefile

```diff
@@ -7,7 +7,7 @@ all: upload test
 test:
 	virtualenv env
 	env/bin/pip install -r requirements.txt
-	env/bin/python main.py https://${GCLOUD_PROJECT}.appspot.com/
+	env/bin/python main.py http://${GCLOUD_PROJECT}.appspot.com/
```

Comment:
is it requests/urllib3 screaming? If so, we just need to do `pip install requests[security]` instead of `requests`. ;)